packages/site/index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Atlantis</title>
-  <link rel="icon" type="image/svg+xml" href="./public/atlantis_favicon.svg" />
+  <link rel="icon" type="image/svg+xml" href="/atlantis_favicon.svg" />
```

Comment:
Removed the `./public` to remove this warning in the terminal

<img width="674" alt="Screenshot 2024-12-24 at 8 38 54 PM" src="https://github.com/user-attachments/assets/01e74fa3-9c53-45e3-8e63-e6767843cc06" />